stacks-common/src/codec/tests/mod.rs

```diff
@@ -0,0 +1,71 @@
+use std::io::Cursor;
+
+use super::*;
+
+#[test]
+fn codec_for_bool() {
+    let t = true;
+    let f = false;
+    let t_binary = t.serialize_to_vec();
+    let f_binary = f.serialize_to_vec();
+    assert_eq!(&t_binary, &vec![1u8]);
+    assert_eq!(&f_binary, &vec![0u8]);
+    assert_eq!(
+        bool::consensus_deserialize(&mut Cursor::new(&vec![1u8])).unwrap(),
```

Comment:
Could you add an assert where you call deserialize for t_binary and f_binary respectively?

Reply:
Do you mean this?
```rust
    assert_eq!(
        bool::consensus_deserialize(&mut Cursor::new(&vec![1u8])),
        Ok(true)
    );
```

Can you give me an example if it's not?

Reply:
But the only difference is that I'm using literal instead of a variable, isn't it?

Reply:
With the literals, there is still a chance that they could get out of sync if one is changed without changing the other, while this style ensures that the serialize and deserialize are coordinated.

Reply:
Understood. I do prefer defining inputs and expected results above all, but if this is the convention, I'd like to stick to it. Please check again.